boards/st/stm32u083c_dk/doc/index.rst

```diff
@@ -0,0 +1,284 @@
+  - Hardware protection feature (HDP)
+  - Secure boot
+  - AES: 128/256-bit key encryption hardware accelerator
+  - True random number generation, candidate for NIST SP 800-90B certification
+  - Candidate for Arm® PSA level 1 and SESIP level 3 certifications
+  - 5 passive anti-tamper pins
+  - 96-bit unique ID
+
+- Up to 10 timers, 2 watchdogs and RTC:
+
+  - 1x 16-bit advanced motor-control, 1x 32-bit and 3x 16-bit general purpose,
+    2x 16-bit basic, 3x low-power 16-bit timers (available in Stop mode),
+    2x watchdogs, SysTick timer
+  - RTC with hardware calendar, alarms and calibration
+
+- Up to 20 communication peripherals:
+
+  - 1 USB 2.0 full-speed crystal-less solution with LPM and BCD
+  - 7 USARTs/LPUARTs (SPI, ISO 7816, LIN, IrDA, modem)
+  - 4 I2C interfaces supporting Fast-mode and Fast-mode Plus (up to 1 Mbit/s)
+  - 3 SPIs, plus 4x USARTs in SPI mode
+  - IRTIM (Infrared interface)
+
+- Rich analog peripherals (independent supply):
+
+  - 1x 12-bit ADC (0.4 µs conversion time), up to 16-bit with hardware oversampling
+  - 1x 12-bit DAC output channel, low-power sample and hold
+  - 1x general-purpose operational amplifier with built-in PGA (variable gain up to 16)
+  - 2x ultra-low-power comparators
+
+- LCD driver:
+  - 8*48 or 4*52 segments, with step-up converter
+
+- General-purpose inputs/outputs:
+  - Up to 69 fast I/Os, most of them 5 V-toleranZ
+
+- ECOPACK2 compliant packages
+
+More information about STM32U083MC can be found here:
+
+- `STM32U083MC on www.st.com`_
+- `STM32U083 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32u083c-dk board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/st/nucleo_u083rc/nucleo_u083rc_defconfig`
+
+
+Connections and IOs
+===================
+
+STM32U083C_DK Board has 10 GPIO controllers. These controllers are responsible
+for pin muxing, input/output, pull-up, etc.
+
+For more details please refer to `STM32U083MC User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- ADC1_IN8 : PA4
+- I2C1_SCL : PB8
+- I2C1_SDA : PB9
+- LPUART_1_TX : PG7
+- LPUART_1_RX : PG8
+- SPI1_NSS : PA4
+- SPI1_SCK : PA5
+- SPI1_MISO : PA6
+- SPI1_MOSI : PA7
```

Comment:
These do not yet exist but I guess you these will follow soon so I won't block.
But wouln't it be better to use PB6 or PA15  for SPI1_CS should that SPI peripherals on the arduino connector or mikrobus connector can be used?

Reply:
For the SPI1_CS, i think it would be better to use PA15 because, according to the user manual, the pin PB6 is used by the USB.
![Screenshot from 2024-08-21 21-12-27](https://github.com/user-attachments/assets/6329a534-90f5-4be0-b3a3-88d2d9a0344a)